src/site/content/en/blog/top-cwv-2023/index.md

```diff
@@ -0,0 +1,203 @@
+
+Another critical aspect of prioritizing the LCP resource is to ensure you don't do anything that causes it to be **deprioritized**, such as adding the `loading="lazy"` attribute. Today, [10%](https://almanac.httparchive.org/en/2022/performance#lcp-lazy-loading) of pages actually set `loading="lazy"` on their LCP image. Beware of image optimization solutions that indiscriminately apply lazy-loading behavior to all images. If they provide a way to override that behavior, be sure to use it for the LCP image. If you're not sure which image will be the LCP, try using heuristics to pick a reasonable candidate.
+
+Deferring non-critical resources is another way to effectively boost the relative priority of the LCP resource. For example, scripts that are not powering the user interface (like analytics scripts or social widgets) can be safely postponed until after the `load` event fires, which ensures they won't compete with other critical resources (such as the LCP resource) for network bandwidth.
+
+To summarize, you should follow these best practices to ensure that the LCP resource is loaded early, and at high priority:
+
+- **Add `fetchpriority="high"` to the `<img>` tag of your LCP image.** If the LCP resource is loaded via a` <link rel="preload">` tag, fear not because you can also set `fetchpriority="high"` on that!\
+- **Never set `loading="lazy"` on the `<img>` tag of your LCP image.** Doing this will deprioritize your image and delay when it starts loading.\
+- **Defer non-critical resources when possible.** Either by moving them to the end of your document, or loading them asynchronously via script.
+
+### Use a CDN to optimize document and resource TTFB
+
+The previous two recommendations focused on making sure your LCP resource is discovered early and prioritized so it can start loading right away. The final piece to this puzzle is making sure the initial document response arrives as quickly as possible too.
+
+The browser cannot start loading any subresources until it receives the first byte of the initial HTML document response, and the sooner that happens, the sooner everything else can start happening as well.
+
+This time is known as [Time to First Byte (TTFB)](/ttfb/), and the best way to reduce TTFB is to:
+
+- Serve your content as geographically close to your users as possible
+- Cache that content so recently-requested content can be served again quickly.
+
+The best way to do both of these things is to [use a CDN](/content-delivery-networks/). CDNs distribute your resources to [edge servers](https://en.wikipedia.org/wiki/Edge_computing), which are spread across the globe, thus limiting the distance those resources have to travel over the wire to your users. CDNs also usually have fine-grained caching controls that can be customized and optimized for your site's needs.
```

Comment:
Not sure this link it really right as not just CDNs. Plus edge computing could be confused with edge workers, that you reference later.

```suggestion
The best way to do both of these things is to [use a CDN](/content-delivery-networks/). CDNs distribute your resources to edge servers, which are spread across the globe, thus limiting the distance those resources have to travel over the wire to your users. CDNs also usually have fine-grained caching controls that can be customized and optimized for your site's needs.
```

Reply:
Done